code/__HELPERS/unsorted.dm

```diff
@@ -256,8 +256,8 @@
 	var/list/sorted_output = list()
 	var/list/sortedplayers = list()
 	var/list/sortedmobs = list()
-	for(var/mob/M in mob_list) //divide every mob into either players (has a mind) or non-players (no mind). braindead/catatonic/etc. mobs included in players
-		if(isnull(M))
+	for(var/mob/M in mob_list) //Divide every mob into either players (has a mind) or non-players (no mind). Braindead/catatonic/etc. mobs included in players
+		if(isnull(M) || M.z == 0) //Ignore null entries or anything in nullspace
```

Comment:
I don't remember whether or not this will cause problems for mobs inside atoms.

Reply:
You're right, I just tested this and mobs inside atoms are at z == 0. Do you have any suggestions for how to check if the mob is in nullspace? If not, I'll just remove this bit. I mainly wanted to fix the borer thing anyway.

Reply:
Okay, I changed it from
`if(isnull(M) || M.z == 0)`
to
`if(isnull(M) || (!M.loc))`
...and it shows mobs inside atoms, but not mobs in nullspace. Working as intended, I think.

Reply:
This is out of the scope of this PR, but shouldn't we prevent mobs from being nullspaced?

Reply:
I'm not sure how it happens, really. I decided to address nullspaced mobs in this PR because, in the same (live) round I discovered the issue with borers, I also noticed some "Unknown" bodies in the Haunt list that nullspaced me when I tried to observe them.

It wasn't an issue with mobs inside atoms, either, as I was able to observe corpses inside morgue trays just fine. /shrug